sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleEditForm.java

```diff
@@ -29,6 +29,12 @@
 import de.symeda.sormas.api.sample.PathogenTestResultType;
 import de.symeda.sormas.api.sample.SampleDto;
 import de.symeda.sormas.api.utils.fieldaccess.UiFieldAccessCheckers;
+import de.symeda.sormas.ui.utils.FieldHelper;
+
+import java.util.Arrays;
```

Comment:
I think your import settings are wrong; java imports should go above the symeda imports. Have you enabled "Optimize imports on the fly"? If so, that might be the case, it caused IntelliJ to not properly regard our import settings in my case.

Reply:
Seems my IntelliJ just removes this plugin with each major update 🙈